Game/Source/GDKTestGyms/Public/BlackboardValues.h

```diff
@@ -12,11 +12,11 @@ struct GDKTESTGYMS_API FBlackboardValues
 	GENERATED_BODY()
 
 	UPROPERTY()
-	FVector TargetAValue; // Points to run between
+	FVector TargetAValue = FVector(EForceInit::ForceInit); // Points to run between
```

Comment:
Haven't seen EForceInit::ForceInit before - why is it required?

Reply:
So I found this while looking at the FVector constructors. It can either take nothing (being uninitialized), xyz, or this force init enum, which can be ForceInit or ForceInitToZero. So I just chose ForceInit, since it didn't matter to me what it was, but maybe it should be ForceInitToZero?

Reply:
re: original question iirc UE now warns (errors?) if values are not initialised right @mironec ?

Reply:
ah, yes

Reply:
If we have to pick one, might as well ForceToZero, it's not like the performance trade-offs will be noticeable in this instance. Is there a benefit in using ForceToZero over FVector::Zero?

Reply:
whooooops
I merged

Reply:
in the fvector case, pretty sure FVector::Zero and ForceInitToZero is equivalent
however, EForceInit is accepted in constructors elsewhere as well, such as color, box, etc.